### PR TITLE
MSC2630: checking public keys in SAS verification

### PR DESCRIPTION
**Pull request by uhoreg**
[Rendered](https://github.com/uhoreg/matrix-doc/blob/sas2/proposals/2630-sas-check-public-keys.md)

### CODE REVIEW

File: proposals/2630-sas-check-public-keys.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# checking public keys in SAS verification
```
Comment (ara4n) [inline proposals/2630-sas-check-public-keys.md:1]:
```suggestion
# Checking public keys in SAS verification
```

File: proposals/2630-sas-check-public-keys.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+## Implementation
+
+This has been implemented in:
+
+- Riot Web 1.6.3 (matrix-js-sdk 6.2.0)
+- Riot Android 0.9.12 (matrix-android-sdk 0.9.35)
+- RiotX 0.21
+- Riot iOS 0.11.5 (matrix-ios-sdk 0.16.5)
+- matrix-weechat and pantalaimon (matrix-nio 0.12.0)
+- FluffyChat (famedlysdk)
```
Comment (Sorunome) [inline proposals/2630-sas-check-public-keys.md:55]: That's actually not merged yet, still gotta write tests for it! (The public builds don't have any emoji verification at all yet)

File: proposals/2630-sas-check-public-keys.md (hunk 1 above)
Comment (uhoreg) [reply proposals/2630-sas-check-public-keys.md:1]: /me counters with

```suggestion
# MSC2630: Checking public keys in SAS verification
```

**Comment by ara4n** [comment]
this got an OOB lgtm from @mimoo :)

**Comment by uhoreg** [comment]
@mscbot fcp merge

**Comment by richvdh** [comment]
I'll assume that others have considered the cryptographic aspects of this, looks fine from a spec POV

**Comment by uhoreg** [comment]
Merged! :tada:

**Pull request merged**